core/src/main/scala/cats/instances/stream.scala

```diff
@@ -140,6 +140,8 @@ trait StreamInstances extends cats.kernel.instances.StreamInstances {
 
       override def toList[A](fa: Stream[A]): List[A] = fa.toList
 
+      def toIterable[A](fa: Stream[A]): Iterable[A] = fa
```

Comment:
I don't know how to override this here since `toIterable` is not part of the `Foldable` typeClass, any suggestions?

Reply:
@travisbrown just to confirm again, are we good here?

Reply:
Oh, hmm. I think we should just remove it entirely, since it won't be part of the public interface anyway.

Reply:
okay, thanks.

Reply:
just curious though, if `toIterable` didn't have a default implementation how would have we implemented it then (in a binary compatible way). 

It looks like we would have had to introduce another typeclass in that case?

Reply:
It's pretty rare to add methods to type classes that don't have default implementations (and this would break bincompat even on 2.12+), so fortunately the question doesn't come up much. But yeah, I think that's what we'd have to do if it did come up somehow.